templates/resources/fabric_connection.md.tmpl

```diff
@@ -149,7 +157,7 @@ Optional:
 
 Optional:
 
-- `authentication_key` (String) Authentication key for provider based connections
+- `authentication_key` (String) Authentication key for provider based connections or Metal NIMF connections
```

Comment:
While you're in here; instead of manually updating this I would recommend using {{ .SchemaMarkdown }} to replace the attributes list and have it build the correct attribute from the description in the schema.